package.json

```diff
@@ -37,7 +37,7 @@
   "bugs": "https://github.com/aurora-is-near/aurora-cli/issues",
   "repository": "https://github.com/aurora-is-near/aurora-cli.git",
   "dependencies": {
-    "@aurora-is-near/engine": "aurora-is-near/aurora.js#400b14c",
+    "@aurora-is-near/engine": "aurora-is-near/aurora.js#2ea6459",
```

Comment:
This should be updated after merge in aurora.js.

Reply:
Done in https://github.com/aurora-is-near/aurora-cli/pull/7/commits/f6b6622b803b641f2ea2317a52c0509130ee69de